.github/workflows/run-kubecover-api-manager.yaml

```diff
@@ -0,0 +1,23 @@
+name: API Manager Kubecover Flow
+
+on: [pull_request]
+
+jobs:
+  kubecover:
+    uses: storageos/api-manager/.github/workflows/kubecover-actions-flow.yaml@master
+    with:
+      platform_builder_version: "stable"
+      kubecover_version: "latest"
+      k8s_version: '1.22'
+      operator_version: 'v2.5.0'
```

Comment:
As of this morning `operator_version` and `node_container` are out of date. `v2.6.0` is the latest.

Reply:
Good point, I shall update